dateparser/date.py

```diff
@@ -1,4 +1,5 @@
 import collections
+from collections import Set
```

Comment:
> DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.9 it will stop working